src/infra/email/nodemailer.js

```diff
@@ -1,16 +1,34 @@
 import nodemailer from 'nodemailer';
+import hbs from 'nodemailer-express-handlebars';
+import path from 'path';
+
+import dotenv from 'dotenv';
+dotenv.config();
 
 const transporter = nodemailer.createTransport({
   host: process.env.EMAIL_HOST,
   port: process.env.EMAIL_PORT,
   secure: false,
+  ignoreTLS: true,
   service: process.env.EMAIL_SERVICE,
   auth: {
     user: process.env.EMAIL_USER,
     pass: process.env.EMAIL_PASS,
   },
 });
 
+const handlebarOptions = {
```

Comment:
Nomear constante com snake case all caps e colocá-la logo após as importações.

```suggestion
const HANDLEBARS_OPTIONS = {
```